lib/Basic/Version.cpp

```diff
@@ -314,7 +314,7 @@ Optional<Version> Version::getEffectiveLanguageVersion() const {
   // a version newer than any released 4.x compiler, and the
   // "5" case should start returning getCurrentLanguageVersion. We should
   // also check for the presence of SWIFT_VERSION_PATCHLEVEL, and if that's
-  // set apply it to the "3" case, so that Swift 4.0.1 will automatically
+  // set apply it to the "3" case, so that Swift 4 will automatically
```

Comment:
@jrose-apple should this read 3?

Reply:
The comment is correct as originally written. It's saying that under "-swift-version 3" patch levels will propagate onto the compatibility version. 4.0.1 was just an example; it's not a reference to the current version.